cordova-lib/src/cordova/metadata/android_parser.js

```diff
@@ -31,14 +31,20 @@ var fs            = require('fs'),
 
 
 function android_parser(project) {
-    if (!fs.existsSync(path.join(project, 'AndroidManifest.xml'))) {
+    if (!fs.existsSync(path.join(project, 'build.gradle'))) {
         throw new CordovaError('The provided path "' + project + '" is not an Android project.');
     }
 
     // Call the base class constructor
     Parser.call(this, 'android', project);
 
     this.path = project;
+
+    //If we have a gradle file but not a manifest, this is an Android Studio project
+    if(!fs.existsSync(path.join(project, "AndroidManifest.xml"))) {
```

Comment:
@infil00p Apparently this fails the jslint step because of double quotes here, which is why the Travis and Appveyor builds failed. It wants all JS strings to use single quotes.